### DISCUSSION TITLE
How to translate Laravel Nova files?

### DISCUSSION THREAD

**Discussion by sajaddp** [Laravel-Lang/common#22] [Q&A] [answered]
Hi,

Where are the Laravel Nova files located?

**Comment by andrey-helldar** [comment, marked as answer]
Depends on what kind of files you want to find.

Nova files are installed in the application along the path:
`lang/vendor/nova/{locale}.json` for Laravel 8-9
`resources/lang/vendor/nova/{locale}.json` for Laravel < 8

As part of the Laravel Lang project, we store Nova source files here: https://github.com/Laravel-Lang/lang/tree/main/source/packages/nova

We save all translations of keys from json files into a single file. For example, https://github.com/Laravel-Lang/lang/blob/main/locales/fr/fr.json
